chglog_test.go

```diff
@@ -461,3 +461,77 @@ func TestGeneratorWithTagFiler(t *testing.T) {
 [Unreleased]: https://github.com/git-chglog/git-chglog/compare/v1.0.0...HEAD`, strings.TrimSpace(buf.String()))
 
 }
+
+func TestGeneratorWithTimmedBody(t *testing.T) {
```

Comment:
Thank you for this. It was very helpful in demonstrating and testing the feature.